partcad/src/partcad/part_factory_brep.py

```
@@ -0,0 +1,41 @@
+from OCP.BRep import BRep_Builder
+from OCP.BRepTools import BRepTools
+from OCP.TopoDS import TopoDS_Shape
+
+from .part_factory_file import PartFactoryFile
+from . import logging as pc_logging
+
+
+class PartFactoryBrep(PartFactoryFile):
+    def __init__(self, ctx, source_project, target_project, config):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)